integration_test/azure-pipelines/windows-robot-integration-test.yml

```diff
@@ -7,21 +7,34 @@
 # SPDX-License-Identifier: BSD-2-Clause-Patent
 ##
 
+parameters:
+- name: PythonVersionList
+  type: object
+  default:
+    # Job Titles cannot have periods, so we iterate
+    # With a map using entry.key and entry.value
+    '3_9': '3.9'
```

Comment:
Can you define these in one place?

There are multiple options:

- Define in variables / variable group in the ADO pipeline UI
- Define in a single file and bring it in as a template
- Etc.

Reply:
As we are only running integration tests on the pipeline, there is only one location on the pipeline yaml to change and one location on the github action to change. resolving this as won't fix.